db/init_db.js

```diff
@@ -15,6 +15,7 @@ async function buildTables() {
     console.log("Dropping all tables...");
     await client.query(`
     DROP TABLE IF EXISTS reviews ;
+    DROP TABLE IF EXISTS orderProducts;
```

Comment:
make sure orderProducts is named the same as the table you created, so either change orderDetails to orderProducts or vise versa